src/daylight-savings/list.ts

```diff
@@ -19,30 +19,41 @@ const dstTransitionDates: DaylightSavingsRanges = {
     2022: ['2022-03-13', '2022-11-05'],
     2023: ['2023-03-12', '2023-11-04'],
     2024: ['2024-03-10', '2024-11-02'],
+    2025: ['2025-03-09', '2025-11-02'],
+    2026: ['2026-03-08', '2026-11-01'],
+    2027: ['2027-03-14', '2027-11-07'],
   },
   Canada: {
     2021: ['2021-03-14', '2021-11-07'],
     2022: ['2022-03-13', '2022-11-05'],
     2023: ['2023-03-12', '2023-11-04'],
     2024: ['2024-03-10', '2024-11-02'],
+    2025: ['2025-03-08', '2025-11-01'],
+    2026: ['2026-03-14', '2026-11-07'],
```

Comment:
as you added 2027 for usa, you wanna add for all? asking chatgpt will be fastest way here

Reply:
give this payload and ask it to fill for 2027 for all countries. it will give you back full json which you can copy paste and verify!